### ISSUE TITLE
Joi API Documentation Load Failure in Firefox

### ISSUE THREAD

**Issue by mgreenw** [hapijs/hapi.dev#259]
#### Support plan

* *which support plan is this issue covered by?* (e.g. Community, Core, Plus, or Enterprise): Community
* *is this issue currently blocking your project?* (yes/no): no
* *is this issue affecting a production system?* (yes/no): no

#### Context

* *node version*: N/A
* *module version with issue*: hapi.dev 1.0.0
* *last module version without issue*: Unknown
* *environment* (e.g. node, browser, native): Browser
* *used with* (e.g. hapi application, another framework, standalone, ...): Firefox
* *any other relevant information*: Works in Chrome

#### What are you trying to achieve or the steps to reproduce?

I am trying to view the `joi` API Documentation for v15.1.1 in Firefox.

I am able to view the latest version's docs (17.0.2). However, when I go to the page for the `joi` 15.1.1 docs using the version selector dropdown, the page remains on the documentation for 17.0.2.

#### What was the result you got?

In the console, I see the following:

```
SyntaxError: invalid regexp group 8cfc73afe0fd4c233e07.js:1:1
ChunkLoadError: "Loading chunk 17 failed.
(missing: https://hapi.dev/_nuxt/8cfc73afe0fd4c233e07.js)"
    NuxtJS 49
SyntaxError: invalid regexp group 8cfc73afe0fd4c233e07.js:1:1
```

I cannot reproduce this issue for API docs other than `joi`.

**Comment by jarrodyellets** [comment]
Looking into it now

**Comment by jarrodyellets** [comment]
Fixed: #260  The issue has been fixed.  There was a recent feature added that allowed you to import a joi schema from the API to the schema tester.  Unfortunately, that feature included a regex with a positive lookbehind, which firefox doesn't like.  I temporarily removed that feature while I work on a permanent solution.

**Comment by mgreenw** [comment]
@jarrodyellets Thanks!
